src/kv/idbstore.js

```diff
@@ -0,0 +1,38 @@
+// @ts-check
+
+const RELAXED = {durability: "relaxed"};
+const OBJECT_STORE = "chunks";
+
+/**
+ * @param {IDBDatabase} idb
+ * @return {IDBTransaction}
+ */
+export function readTransaction(idb) {
+  return idb.transaction(OBJECT_STORE, "readonly");
```

Comment:
Since "readonly" is only in one place I left it as a string literal.

I think V8 would inline it but 🤷